### PR TITLE
plugins.galatasaraytv: Add support for GALATASARAY SK TV live streams 

### PR DESCRIPTION
**Pull request by yavuzKomecoglu**
Added free live stream [GALATASARAY SK](https://galatasaray.com/) support.
Tested. 360p (worst), 480p, 720p (best) quality is also broadcast continuously.

### CODE REVIEW

File: src/streamlink/plugins/canalplus.py [hunk 1]
```diff
@@ -55,9 +55,9 @@ def _get_streams(self):
         parsed = []
         headers = {'User-Agent': self._user_agent}
 
-        # Some videos may be also available on Dailymotion (especially on CNews)
+        # Some videos may be also available on  (especially on CNews)
         if videos['ID_DM'] != '':
-            for stream in self.session.streams('https://www.dailymotion.com/video/' + videos['ID_DM']).items():
+            for stream in self.session.streams('https://www..com/video/' + videos['ID_DM']).items():
```
Comment (bastimeyer) [inline src/streamlink/plugins/canalplus.py]: This looks like you've accidentially replaced the string "DailyMotion" with "" in the entire project.

Comment (yavuzKomecoglu) [reply src/streamlink/plugins/canalplus.py]: Yes, a single file was accidentally modified. Sorry. Thanks for the fix.

**Comment by yavuzKomecoglu** [comment]
@bastimeyer What are my obligations to be a merge? will an action be taken?

File: src/streamlink/plugins/canalplus.py (hunk 1 above)
Comment (bastimeyer) [reply src/streamlink/plugins/canalplus.py]: Which fix? I've requested a change.

Comment (yavuzKomecoglu) [reply src/streamlink/plugins/canalplus.py]: I undoed the change made by mistake.

**Pull request merged**